src/index.spec.ts

```diff
@@ -1,12 +1,175 @@
+import { enableFetchMocks } from 'jest-fetch-mock';
+import fetchMock from 'jest-fetch-mock';
+enableFetchMocks();
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
-
 test('Testing time!', () => {
   console.log('welcome');
 });
+
+describe('isInteger function test', () => {
```

Comment:
for more readability when you define a describe block only put the name of the function to be tested, for this case should be `describe('isInteger', () => {`